test/unit/postprocessing/conftest.py

```diff
@@ -113,6 +113,14 @@ def predict(self, X):
         return self.scores
 
 
+class PassThroughPredictor(BaseEstimator, ClassifierMixin):
+    def fit(self, X, y=None, **kwargs):
+        return self
+
+    def predict(self, X):
+        return X[0]
+
+
```

Comment:
I think all these would much better fit in the test file themselves instead of in the `conftest.py`. I find it odd to have these here.

Reply:
Perhaps this continues an odd pattern we've accidentally established...
I agree with @adrinjalali, though, and will make a mental note to clean other similar cases up.

Reply:
Done.